pkg/models/html.go

```diff
@@ -0,0 +1,78 @@
+package models
+
+import (
+	"context"
+	"encoding/json"
+	"errors"
+	"fmt"
+	"html/template"
+	"io"
+
+	"github.com/cmsgov/easi-app/pkg/sanitization"
+)
+
+// HTML represents html code. It is sanitized to only allow specific tags
+type HTML string
```

Comment:
It bugs me that defining `HTML` this way means that code elsewhere in the backend could use this type to create an `HTML` value with whatever they want, without sanitizing it, but I don't think there's a way around it. If you made the type private to this module, I assume `gqlgen` couldn't pick it up.

Reply:
Potential alternate design (that's probably more work than it's worth, and I'm not sure if it'd even work):
* Define a GQL scalar named `UnsafeHTML`, `RawHTML`, or similar that's _not_ sanitized, but that shouldn't get used in any backend code unless someone _really_ knows what they're doing.
* Define a `safeHTML` type in a separate module, private to that module.
    * Have a function `func SanitizeHTML(raw RawHTML) (safeHTML, error)` that's the only way to produce `safeHTML` values. 
    * Add public/exported methods on `safeHTML` that allow converting it to a string, a `template.HTML` value, or whatever else is needed.

As I think about this, I don't think it's possible, because if the `safeHTML` type isn't exported, it couldn't be used as a field on models in other packages. (Unless you did something funky by having a struct wrapper around it? But that'd probably cause even _more_ problems) If you can think of a way to get something like this to work, though, it might be worthwhile.

Reply:
Thanks Dylan! These are good thoughts. Per our conversation off line, I'm going to hold off on changing the implementation of this now, but I do like the idea of sanitizing agin when we convert it to Template.